base/show.jl

```diff
@@ -1843,6 +1843,7 @@ function summary(io::IO, a, inds::Tuple{Vararg{OneTo}})
     showarg(io, a, true)
 end
 function summary(io::IO, a, inds)
```

Comment:
Does anybody know why these are methods of `summary`? Looks like it should be a separate internal function, e.g. `array_summary`.

Reply:
No really good reason. #23389 redesigned `summary` fairly substantially, including the `io` argument, and perhaps I was thinking about overrides via the axes range type being part of the public interface.

Reply:
As suggested, now changed to internal function `array_summary`